scale/docs/rest/job.rst

```diff
@@ -1179,3 +1179,260 @@ These services provide access to information about "all", "currently running" an
+| **NOTE**                                                                                                                  |
+|                This API endpoint is available starting with API **v6**.  It replaces a very similar API endpoint which    |
+|                you can see described here: :ref:`Job Execution List <rest_job_execution_details>`.                        |
++---------------------------------------------------------------------------------------------------------------------------+
+| **GET** /jobs/{id}/executions/{exe_num}                                                                                   |
+|         Where {id} is the unique identifier of an existing job and {exe_num} is the execution number of a job execution   |
+|         as it relates to the job.                                                                                         |
++----------------------+----------------------------------------------------------------------------------------------------+
+| **Successful Response**                                                                                                   |
++----------------------+----------------------------------------------------------------------------------------------------+
+| **Status**           | 200 OK                                                                                             |
++----------------------+----------------------------------------------------------------------------------------------------+
+| **Content Type**     | *application/json*                                                                                 |
++----------------------+----------------------------------------------------------------------------------------------------+
+| **JSON Fields**                                                                                                           |
++----------------------+-------------------+--------------------------------------------------------------------------------+
+| id                   | Integer           | The unique identifier of the model. Can be passed to the details API call.     |
+|                      |                   | (See :ref:`Job Execution Details <rest_job_execution_details>`)                |
++----------------------+-------------------+--------------------------------------------------------------------------------+
+| status               | String            | The status of the job execution.                                               |
+|                      |                   | Choices: [RUNNING, FAILED, COMPLETED, CANCELED].                               |
++----------------------+-------------------+--------------------------------------------------------------------------------+
+| command_arguments    | String            | The argument string to execute on the command line for this job execution.     |
```

Comment:
1. command_arguments and environment should be removed (in the field listing and in the example JSON)
2. exe_num and cluster_id should be added to the example JSON and field listing
3. results_manifest should be removed from both the field listing and example JSON
4. results should be renamed output for both field listing and example JSON with a description saying its a JSON describing the job output. There should be a JSON schema for this to link to.
5. The link for configuration should point to the execution configuration, not the job configuration.
6. The node section should not have port and slave_id.
7. task_results needs to be added to field listing and example JSON.
8. last_modified should be removed (in the field listing and in the example JSON)
9. Re-order fields to match the serializer (the same order that will be returned for real)

Reply:
These same changes need applied to the serializers.